crates/air-lib/test-utils/src/test_runner.rs

```diff
@@ -77,6 +99,14 @@ impl<R: AirRunner> TestRunner<R> {
         let mut call_results = HashMap::new();
         let mut next_peer_pks = HashSet::new();
 
+        let key_pair = override_current_peer_id
```

Comment:
`let key_pair = peer_id.map( pair.1 ).or(key_secret)`

the naming is hard to follow, really. Is it `secret` or `key pair`? or `peer_id`? is `pair` == `keypair` or `tuple`?

Reply:
Renamed consistently to `keypair`, `keypair_format`, `keypair_data`.

Reply:
So AVM will have access to both Public and Private key as opposed to just Private key.

If that aligns with the goals, then it's great! :)

Reply:
Yes, that's what the goal is, as it puts its key into data.